### ISSUE TITLE
Can't push to remote

### ISSUE THREAD

**Issue by japanese-goblinn** [git-up/GitUp#893]
Hi, love GitUp so far but I can't manage to understand why push is not working, while fetch is working fine. Also other Fork (git client) and CLI are working fine. Where should I look into to fix this issue?

<img width="372" alt="image" src="https://user-images.githubusercontent.com/33631005/208264332-6348b0ec-edc5-4789-9d6e-caa35312b2b0.png">

**Comment by lucasderraugh** [comment]
Haven't seen this issue yet. Anything special and is this a regression for you from 1.3.2?

**Comment by japanese-goblinn** [comment]
@lucasderraugh I never used GitUp before and 1.3.4 is my first version. Also I managed to find that error happening in the `git_remote_connect_ext`(1 image) function after prompt for keychain access (2 image)

<img width="739" alt="image" src="https://user-images.githubusercontent.com/33631005/208268553-7fcced72-7a04-451b-86d3-9f20a73f38bc.png">

<img width="546" alt="image" src="https://user-images.githubusercontent.com/33631005/208268578-e5fdbbc0-5a3c-4d02-8b84-1579442c0421.png">

**Comment by japanese-goblinn** [comment]
@lucasderraugh Oh, okay I figured this out. After pressing`Deny` i field in my username and access token and now it's working! Maybe this is something with two step authentication 🤔?

**Comment by japanese-goblinn** [comment]
As I mentioned in message above when I entering my username and access token everything works but this pop up is keeping showing every time? I started to figuring this out and found out that in `KeychainAccessor savePlainTextAuthenticationToKeychainForURL:` calling of `SecKeychainSetUserInteractionAllowed` returning no error while call of `SecKeychainAddInternetPassword` resulting to `-25299` which according to [this](https://www.osstatus.com/search/results?platform=all&framework=all&search=-25299) site means `Duplicate Item`. That weird and not explaining why pop up is still showing every time event when this item already exist. And more wierd thing that why then It returning error when asking for keychain password when clearly this item there and this credentials works.

Also in release version that I've installed from website it always shows an error and never asks for keychain contrast to debug mode, even when I deleted the app even of I reinstall the app. Any ideas how to reset this?
